internal/cmd/db_replicate.go

```diff
@@ -63,6 +63,10 @@ var replicateCmd = &cobra.Command{
 			return err
 		}
 		dbSettings := config.GetDatabaseSettings(database.ID)
+		if dbSettings == nil {
+			// Backwards compatibility with old settings files.
+			dbSettings = config.GetDatabaseSettings(database.Name)
+		}
```

Comment:
We should still check for nil in case the db is just not in the settings, no?

Reply:
Sure, but the same bug exists in `getUrl()` so let's fix it in a follow up